configs/setups/awicm3/awicm3.yaml

```diff
@@ -952,23 +1130,30 @@ computer:
         merge_component_envs:
             compile: True
 
-        # MA: Only used for JUWELS right now
-        OpenMP_flag: "-qopenmp"
-        OpenMP_CCDEFS: "_OPENMP"
-
-        choose_computer.name:
-            juwels:
-                iolibraries: awi_libs
-                compiler_mpi: intel2022_ompi2022
-            blogin:
-                iolibraries: geomar_libs
-                compiler_mpi: intel2019_ompi
-            glogin:
-                iolibraries: geomar_libs
-                compiler_mpi: intel2019_ompi
-            levante:
-                iolibraries: system_libs
-                compiler_mpi: intel2022_openmpi
+        choose_general.major_version:
```

Comment:
I think there is some redundancy here with the oifs envs yamls. Now conflicts will be catch, but it might be worth it have it them in either one place or another.